libobs/util/platform-cocoa.m

```diff
@@ -81,6 +81,14 @@ int os_get_config_path(char *dst, size_t size, const char *name)
 	return snprintf(dst, size, "%s/%s", base_path, name);
 }
 
+int os_get_data_path(char *dst, size_t size, const char *name) {
```

Comment:
All new functions seem to violate the coding style: opening braces for functions belong on a new line, e.g.:
```cpp
int os_get_data_path(char *dst, size_t size, const char *name)
{
```